src/test/java/net/sf/jabref/logic/util/strings/StringUtilTest.java

```diff
@@ -94,20 +94,10 @@ public void testJoin() {
     public void testStripBrackets() {
         assertEquals("foo", StringUtil.stripBrackets("[foo]"));
         assertEquals("[foo]", StringUtil.stripBrackets("[[foo]]"));
-        assertEquals("foo", StringUtil.stripBrackets("foo]"));
```

Comment:
Could you keep the line as `assertNotEquals`? - The documentation in the tests should be self-contained and not rely on the discussion we do on GitHub ^^